example/ios/Flutter/flutter_export_environment.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 # This is a generated file; do not edit or check into version control.
-export "FLUTTER_ROOT=/Users/lizhuoyuan/fvm/versions/stable"
-export "FLUTTER_APPLICATION_PATH=/Users/lizhuoyuan/Development/Project/flutter_screenutil/example"
+export "FLUTTER_ROOT=/home/juzi/flutter"
+export "FLUTTER_APPLICATION_PATH=/home/juzi/workspaces/flutter_screenutil/example"
```

Comment:
Hi, I think this change should be reverted back.

Reply:
reverted.